bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/InputDataFormattingTest.groovy

```diff
@@ -441,62 +431,62 @@ class InputDataFormattingTest {
             assertNotNull(row.CROSSING)
             assertNotEquals('', row.CROSSING)
         }
-        /*
+        
         // Specific cases
         // -------------------------------
-        //... with the road 'TRONROUT0000000306711343' : LARGEUR = 0 / NATURE = 'Sentier'
+        //... with the road 'ROUTE0001' : LARGEUR = 0 / NATURE = 'Sentier'
         assertEquals(1, h2GISDatabase.firstRow("SELECT WIDTH FROM ROAD " +
-                "WHERE ID_SOURCE='TRONROUT0000000306711343';")["WIDTH"])
+                "WHERE ID_SOURCE='ROUTE0001';")["WIDTH"])
         assertEquals('path', h2GISDatabase.firstRow("SELECT TYPE FROM ROAD " +
-                "WHERE ID_SOURCE='TRONROUT0000000306711343';")["TYPE"])
+                "WHERE ID_SOURCE='ROUTE0001';")["TYPE"])
 
-        //... with the road 'TRONROUT0000000087744143' : LARGEUR = 5,5 / NATURE = 'Route à 1 chaussée'
+        //... with the road 'ROUTE0002' : LARGEUR = 5,5 / NATURE = 'Route à 1 chaussée'
         assertEquals(5.5, h2GISDatabase.firstRow("SELECT WIDTH FROM ROAD " +
-                "WHERE ID_SOURCE='TRONROUT0000000087744143';")["WIDTH"])
+                "WHERE ID_SOURCE='ROUTE0002';")["WIDTH"])
         assertEquals('unclassified', h2GISDatabase.firstRow("SELECT TYPE FROM ROAD " +
-                "WHERE ID_SOURCE='TRONROUT0000000087744143';")["TYPE"])
+                "WHERE ID_SOURCE='ROUTE0002';")["TYPE"])
 
-        //... with the road 'TRONROUT0000000306712994' : LARGEUR = 0 / NATURE = 'Route empierrée'
+        //... with the road 'ROUTE0003' : LARGEUR = 0 / NATURE = 'Route empierrée'
```

Comment:
Idem group all values for the same features

Reply:
Done